pandahub/lib/PandaHub.py

```diff
@@ -1775,12 +1795,17 @@ def get_variant_filter(self, variant: int | None) -> dict:
         dict
             mongodb query filter for the given variant
         """
-        validate_variant_type(variant)
```

Comment:
What about this function? Was is deleted or deprecated?

Reply:
`validate_variant_type` got wrapped in the class method `validate_variant`, which additionally checks if the (optionally passed) `element_type` is not contained in `self._elements_without_vars`, a list of element types which do not use the variant-schema (var_type / not_in_var / variant) - see code below.

It might actually be better to use whitelisting here instead, i.e. keep a list of elements which *do* use the schema.